<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThwbWindow</class>
 <widget class="QMainWindow" name="ThwbWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>44</y>
      <width>132</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>48</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>查找：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>46</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>替换成：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>12</y>
      <width>106</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>替换文本</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>添加文本</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>格式</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>79</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>示例：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rename">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>73</y>
      <width>70</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>重命名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>46</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>79</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_openfd">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>73</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件夹</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
